### ISSUE TITLE
How to store/pass session ID between hooks?

### ISSUE THREAD

**Issue by mmalecki** [Relief-1/Relief1#20]
Problem: session manager needs some session ID (unique for each user), by which he could query Riak. However, AFAIK, i  socket.io/hook.io there's no way to get access to current request context (so that I could extract this token anytime I want).

Only way I see now is make every API function which needs session access have `sessionID` parameter. For me, it's -1, in most cases we'll have really long call chain and it may get nasty.

Any ideas how to get rid of that?

**tl;dr:** No `sessionID` parameter in almost every function call, how?

/cc @AvianFlu
